lec09-Randomization-tests.Rmd

```diff
@@ -195,17 +205,26 @@ Note the differences in body width values between the original and reshuffled da
 # Plot histograms of reshuffled data
 ggplot(reshuffled, aes(x = width)) +
     ylab("Count") + xlab("Body width") +
-    geom_histogram(data = reshuffled[reshuffled$sex == "Male",], aes(x = width), bins = 30, fill = "red", alpha = 0.4, colour = "black") +
-    geom_histogram(data = reshuffled[reshuffled$sex == "Female",], aes(x = width), bins = 30, fill = "blue", alpha = 0.4, colour = "black") +
-    geom_vline(data = reshuffled[reshuffled$sex == "Male",], aes(xintercept = mean(width)), size = 1, linetype = "dashed", colour = "red") +
-    geom_vline(data = reshuffled[reshuffled$sex == "Female",], aes(xintercept = mean(width)), size = 1, linetype = "dashed", colour = "blue") +
+    geom_histogram(data = reshuffled[reshuffled$sex == "Male",], 
+                   aes(x = width), bins = 30, fill = "red", 
+                   alpha = 0.4, colour = "black") +
+    geom_histogram(data = reshuffled[reshuffled$sex == "Female",], 
+                   aes(x = width), bins = 30, fill = "blue", 
+                   alpha = 0.4, colour = "black") +
+    geom_vline(data = reshuffled[reshuffled$sex == "Male",], 
+               aes(xintercept = mean(width)), size = 1, 
+               linetype = "dashed", colour = "red") +
+    geom_vline(data = reshuffled[reshuffled$sex == "Female",], 
+               aes(xintercept = mean(width)), size = 1, 
+               linetype = "dashed", colour = "blue") +
```

Comment:
This is even more set up to use `fill = Sex`, since `reshuffled` is already one dataset! Why not use that instead of this more complicated code?